localboot/grub.go

```diff
@@ -65,6 +76,9 @@ func ParseGrubCfg(grubcfg string, basedir string, grubVersion int) []bootconfig.
 			}
 			inMenuEntry = true
 			cfg = new(bootconfig.BootConfig)
+			name := strings.Join(sline[1:], " ")
```

Comment:
this will panic on malformed lines, check that its length is at least 2 before this

Reply:
done

Reply:
done